dev/build.image/build.gradle

```diff
@@ -884,4 +897,4 @@ if (isAutomatedBuild && !isIFIXBuild) {
         }
         publish.dependsOn buildOsNativePackages
     }
-}
```

Comment:
remove this change too or add a new line at the end of the file, I think adding a new line at the end is common practice.